doc/tutorial/remote-apis.md

```diff
@@ -0,0 +1,104 @@
+# Remote APIs
+
+JSTP allows you to use remote APIs as regular asynchronous functions, let's
+examine how it looks like by updating our `client.js` file like this(as this
+updated piece of code works in Node.js as well as in browser, we will consider
```

Comment:
Yes, this code will work without any modifications in browser, so in order to avoid duplicating in the next sections, I want to explain this here. Maybe, this formulation isn't clear enough.

Reply:
I'm not sure about better explaining, so haven't fixed this yet

Reply:
Have you tried running it in a browser?

Reply:
Sure, everything works

Reply:
Sure, everything works

Reply:
Should I add example for browser?